tests/test_core.py

```diff
@@ -1,3 +1,4 @@
+from ctypes.wintypes import PINT
```

Comment:
What's this for? Automatically import? 

Make sure before push, check the changes locally.

Reply:
Automatically import.